src/phone_formatter.js

```diff
@@ -241,4 +329,831 @@ class PhoneFormatter extends DelimitedTextFormatter {
   }
 }
 
+const COUNTRY_CODES = [
+  {
```

Comment:
Sounds good.

Overall I think breaking this up wouldn't solve this issue. If I want phone formatters for USA and Mexico, I still end up bringing in rest of the world. Manually passing just the data that one needs feels like the right approach when majority of default data otherwise could be considered total waste.